### PR TITLE
Run xargs iptables in RD distro

### PR DESCRIPTION
**Pull request by Nino-K**
This change defines a function for wsl with default to rancher-desktop distro. Also, allows the `xargs iptables` call to get triggered inside the rancher-desktop distro.

### CODE REVIEW

File: bats/tests/helpers/commands.bash [hunk 1]
```diff
@@ -80,3 +80,8 @@ rdshell() {
 rdsudo() {
     rdshell sudo "$@"
 }
+WSL_DISTRO=rancher-desktop
```
Comment (jandubois) [inline bats/tests/helpers/commands.bash:83]: So far we have put variable defaults at the top, before all the function definitions (see e.g. `CONTAINERD_NAMESPACE`, which is conceptually similar.

**Review by jandubois — DISMISSED** [review]
Thanks, LGTM.

I would move the `WSL_DISTRO` default to where the `CONTAINERD_NAMESPACE` is set, but leave this up to you.

File: bats/tests/helpers/commands.bash (hunk 1 above)
Comment (Nino-K) [reply bats/tests/helpers/commands.bash:83]: Yeah, to keep things consistent let me move it.

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**